### ISSUE TITLE
Unable to run latest circleci-nightly tagged proxy image

### ISSUE THREAD

**Issue by nixgadget** [istio/issues#341]
Pulled down 0.8.0 pre release build and tried to install via Helm on local machine.
I noticed the two pods kept crashing,
```
istio-policy-5d96b4f444-7qlv2             1/2    CrashLoopBackOff  1         18s
istio-telemetry-6b7f9b7868-2qqwc          1/2    CrashLoopBackOff  1         18s 
```
Looking at the logs the following gets thrown,
```
Error: unknown flag: --address
Usage:
  mixs server [flags]

Flags:
      --adapterWorkerPoolSize int         Max number of goroutines in the adapter worker pool (default 1024)
      --apiWorkerPoolSize int             Max number of goroutines in the API worker pool (default 1024)
      --configDefaultNamespace string     Namespace used to store mesh wide configuration. (default "istio-system")
      --configStoreURL string             URL of the config store. Use k8s://path_to_kubeconfig or fs:// for file system. If path_to_kubeconfig is empty, in-cluster kubeconfig is used.
  -h, --help                              help for server
      --livenessProbeInterval duration    Interval of updating file for the liveness probe.
      --livenessProbePath string          Path to the file for the liveness probe.
      --log_as_json                       Whether to format output as JSON or in plain console-friendly format
      --log_caller string                 Comma-separated list of scopes for which to include caller information, scopes can be any of [default, attributes, adapters]
      --log_output_level string           Comma-separated minimum per-scope logging level of messages to output, in the form of <scope>:<level>,<scope>:<level>,... where scope can be one of [default, attributes, adapters] and level can be one of [debug, info, warn, error, none] (default "default:info")
      --log_rotate string                 The path for the optional rotating log file
      --log_rotate_max_age int            The maximum age in days of a log file beyond which the file is rotated (0 indicates no limit) (default 30)
      --log_rotate_max_backups int        The maximum number of log file backups to keep before older files are deleted (0 indicates no limit) (default 1000)
      --log_rotate_max_size int           The maximum size in megabytes of a log file beyond which the file is rotated (default 104857600)
      --log_stacktrace_level string       Comma-separated minimum per-scope logging level at which stack traces are captured, in the form of <scope>:<level>,<scope:level>,... where scope can be one of [default, attributes, adapters] and level can be one of [debug, info, warn, error, none] (default "default:none")
      --log_target stringArray            The set of paths where to output the log. This can be any path as well as the special values stdout and stderr (default [stdout])
      --maxConcurrentStreams uint         Maximum number of outstanding RPCs per connection (default 1024)
      --maxMessageSize uint               Maximum size of individual gRPC messages (default 1048576)
      --monitoringPort uint16             HTTP port to use for the exposing mixer self-monitoring information (default 9093)
  -p, --port uint16                       TCP port to use for Mixer's gRPC API (default 9091)
      --profile                           Enable profiling via web interface host:port/debug/pprof (default true)
      --readinessProbeInterval duration   Interval of updating file for the readiness probe.
      --readinessProbePath string         Path to the file for the readiness probe.
      --singleThreaded                    If true, each request to Mixer will be executed in a single go routine (useful for debugging)
      --trace_jaeger_url string           URL of Jaeger HTTP collector (example: 'http://jaeger:14268/api/traces?format=jaeger.thrift').
      --trace_log_spans                   Whether or not to log trace spans.
      --trace_zipkin_url string           URL of Zipkin collector (example: 'http://zipkin:9411/api/v1/spans').
```

Looks like `--address` option no longer exists.

**Comment by kyessenov** [comment]
It's a new option. I have seen this happening if you happen to have cached
mixer images. You can force pull with image pull policy always, and that
should update mixer images.

**Comment by nixgadget** [comment]
By the way this was installed on a clean slate minikube vm.

**Comment by kubeway** [comment]
I also encountered this problem

**Comment by douglas-reid** [comment]
This should be resolved in the full 0.8 release. Closing. Feel free to reopen if still experiencing.
